### ISSUE TITLE
Saved profiles covered up by menu preview

### ISSUE THREAD

**Issue by ewscott9** [mozilla/FirefoxColor#1027]
When you save a profile, it gets put into a menu tab on the Web UI that is blocked by the menu preview image. When in full screen on 1080p monitor you can see the saved themes option. But it is blocked on lower resolutions, and in windowed mode.

![window](https://user-images.githubusercontent.com/13879404/178323392-0774c0c6-0171-409e-91a6-004e33a805ab.jpg)
![Full](https://user-images.githubusercontent.com/13879404/178323400-1f17b429-2809-4593-9678-be4715f0bd2d.jpg)

**Comment by FayneAldan** [comment]
Duplicate of #972

**Comment by adiac** [comment]
Well, this is an very old issue, but on Firefox 136.0.4 this bug is still present

![Image](https://github.com/user-attachments/assets/43b1c130-af3e-4f09-8531-16b2a36e595b)
